db/migrations/20220630052212-add-github-client-secrets-to-github-server-apps.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+
+const tableName = "GitHubServerApps";
+
+module.exports = {
+	up: async (queryInterface, Sequelize) => {
+		await queryInterface.addColumn(tableName, "encryptedGitHubClientSecret", {
+			type: Sequelize.TEXT,
+			allowNull: false
+		});
+		await queryInterface.addColumn(tableName, "encryptedWebhookSecret", {
+			type: Sequelize.TEXT,
+			allowNull: false
+		});
+		await queryInterface.addColumn(tableName, "encryptedPrivateKey", {
+			type: Sequelize.TEXT,
+			allowNull: false
+		});
+		await queryInterface.removeColumn(tableName, "secrets");
```

Comment:
this might cause an issue on prod on deployment.  remove this for now, we can delete it once all code refers only to the new columns